addons/calendar/models/calendar_event.py

```diff
@@ -1040,6 +1040,14 @@ def ics_datetime(idate, allday=False):
             for attendee in meeting.attendee_ids:
                 attendee_add = event.add('attendee')
                 attendee_add.value = u'MAILTO:' + (attendee.email or u'')
+
+            # Add "organizer" field if email available
+            if meeting.partner_id and meeting.partner_id.email:
```

Comment:
Actually doing "if meeting.partner_id.email" is sufficient, as partner_id will be a void recordset and email will be Falsy.